src/main/kotlin/org/jetbrains/research/testspark/tools/kex/Kex.kt

```diff
@@ -0,0 +1,130 @@
+package org.jetbrains.research.testspark.tools.kex
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.roots.ProjectRootManager
+import org.jetbrains.research.testspark.actions.controllers.TestGenerationController
+import org.jetbrains.research.testspark.data.CodeType
+import org.jetbrains.research.testspark.data.FragmentToTestData
+import org.jetbrains.research.testspark.langwrappers.PsiHelper
+import org.jetbrains.research.testspark.langwrappers.PsiMethodWrapper
+import org.jetbrains.research.testspark.services.PluginSettingsService
+import org.jetbrains.research.testspark.tools.Pipeline
+import org.jetbrains.research.testspark.tools.kex.generation.KexProcessManager
+import org.jetbrains.research.testspark.tools.template.Tool
+import java.io.File
+
+class Kex(override val name: String = "Kex") : Tool {
+    private val log = Logger.getInstance(this::class.java)
+
+    /**
+     * Returns a new instance of KexProcessManager for the given project.
+     *
+     * @param project The IntelliJ IDEA project for which the KexProcessManager is created.
+     * @return The KexProcessManager instance created for the given project.
+     */
+    private fun getKexProcessManager(project: Project): KexProcessManager {
+        val projectClassPath: String = ProjectRootManager.getInstance(project).contentRoots.first().path
+        val settingsProjectState = project.service<PluginSettingsService>().state
+        val buildPath = "$projectClassPath${File.separatorChar}${settingsProjectState.buildPath}"
```

Comment:
It is better to use `ToolUtils.sep` here, so taht we are consistent with the other code

Reply:
I've switched to using a helper function I made `ToolUtils.osJoin()`. So its not consistent with older code but I hope it gets used from now on